pontoon/base/views.py

```diff
@@ -58,6 +58,10 @@ def home(request):
     user = request.user
     project = Project.objects.get(id=1)
 
+    # If user is not logged in, the default homepage is set to static webpage
+    if not user.is_authenticated():
+        return render(request, 'home.html')
```

Comment:
This means we'll still load the all Pontoon Intro page for authenticated users that don't specify a custom homepage. Instead, we want the homepage to be loaded for everybody (regardless if they are logged in or not), unless they set a custom homepage.

To do that, you need to ommit the condition and replace line `return translate...` with this one. Then, you can also remove the lines which set `project`, `locale` and `path`, since they are no longer needed.